torch/_torch_docs.py

```diff
@@ -3587,6 +3587,54 @@ def merge_dicts(*dicts):
     array([-1,  2,  3])
 """)
 
+add_docstr(torch.frombuffer,
+           r"""
+frombuffer(buffer, dtype=None, count=-1, offset=0, *, requires_grad=False) -> Tensor
+
+Create a 1-dimensional :class:`Tensor` from an object that implements
+the Python buffer protocol.
+
+Interpret :attr:`buffer` as a 1-dimensional buffer of type :attr:`dtype`
+with :attr:`count` elements after the first :attr:`offset` bytes.
+
+The returned tensor and buffer share the same memory. Modifications to
```

Comment:
How does this work if the buffer has a different dtype then what was specified?

Reply:
NumPy seems to interpret the buffer as byte data and writes directly to the bytes, which ay be surprising. It'd be nice to clarify this in the documentation if that's the approach we want to take.

Reply:
It has the same behavior as NumPy. It blindly believes the specified `dtype`, and interprets the data accordingly.